libr/include/r_util/r_str.h

```diff
@@ -33,6 +33,8 @@ typedef struct r_charset_t {
 	Sdb *db_char_to_hex;
 	RCharsetRune *custom_charset;
 	size_t remaining;
+    size_t encode_maxkeylen;
```

Comment:
Use tabs not spaces

Reply:
this is not fixed yet. remove the remaining variable and use tabs instead of spaces please